react/features/base/lib-jitsi-meet/native/RTCPeerConnection.js

```diff
@@ -322,6 +325,23 @@ function _synthesizeIPv6Addresses0(sessionDescription) {
                                             resolve(value);
                                         },
                                         reject);
+                                } else if (
+                                    typeof getIpV6Address === 'function') {
+                                    getIpV6Address(ip).then(
+                                        value => {
+                                            if (value) {
+                                                ips.set(ip, value);
+                                            } else {
+                                                ips.delete(ip);
+                                            }
+                                            resolve(value);
+                                        },
```

Comment:
Could we please extract into a function the bulk of the onFulfilled handlers of the `getaddrinfo` and `getIpV6Address` `Promise`s? Otherwise, the new piece feel a little bit like a duplication.

Reply:
I've tried to do that, but not sure if that's what you meant. Please take a look and feel free to adjust if necessary.